tcpclient/connectionsManagement.go

```diff
@@ -1,30 +1,46 @@
 package tcpclient
 
 import (
-	"sync"
+	"bufio"
 	"fmt"
+	"io"
 	"net"
-	"bufio"
 	"strconv"
+	"sync"
 )
 
-func TcpConnect(id int, host string, port int, wg *sync.WaitGroup) error {
-	fmt.Println("\t runner " + strconv.Itoa(id) + " is initiating a connection")
+func reportConnectionStatus(debugOut io.Writer, statusChannel chan<- Connection, connectionDescription Connection)  {
```

Comment:
because this report is only for debugging, I would rename it to something like `debugReport...` , WDYT?

Reply:
Nops, it is not only for debugging. It may write additional debugging information, but it also reports to the goroutine that prints the summarized status of all connections at each interval

Reply:
👍